pkg/ds/farm.go

```diff
@@ -263,7 +263,7 @@ func (dsf *Farm) handleDSChanges(changes dsstore.WatchedDaemonSets, quitCh <-cha
 	}
 
 	if len(changes.Created) > 0 {
-		dsf.logger.Infof("The following daemon sets have been created:")
+		dsf.logger.Infof("The following %d daemon sets have been created:", len(changes.Created))
```

Comment:
Hmm, interesting idea to just give a list of the daemon set IDs in this message, just a second.